### DISCUSSION TITLE
Build a STELLA-Q: Does not require soldering!

### DISCUSSION THREAD

**Discussion by paulmirel** [STELLA-Landsat/STELLA#6] [Build Notes: How do I make a STELLA?]
https://landsat.gsfc.nasa.gov/stella/stella-q/

**Comment by mchopping** [comment]
Hi Paul,  

I wondered what you thought of the STELLA-Q version I built using a cheap, translucent 12.5" plastic pencil case (https://www.amazon.com/dp/B00FZVPX10, video at https://drive.google.com/file/d/1VLfmOXL5aL77Bml0fB5hZ5DvAGHXE-rl/view?usp=drive_link)? 

I intended to let my students choose between the classic stick and the case versions, so I need to tell them to either bring a 12" ruler to make the classic version, or let me know so I can cut the holes in the cases prior to the next class. I would make a few changes over the `#10` model shown in the video:

1.  Cut a small hole for the USB port and relocate the microcontroller so it's flush with the side (but would it need to be more firmly fixed?);
2. Affix the keyboard to 1/2" solid foam glued stuck to the bottom of the case, so the keys are (just) inside but still  accessible;
3.  Maybe make the sensor port a little wider; and 
4.  Use matte black tape on the exterior to minimize reflection from the case.

If there's something I didn't think of that is obvious to you -- or if there are good reasons to stick with the stick version! -- I'd be very grateful if you could let me know.   

Thanks. 

  Mark

p.s. The page at https://landsat.gsfc.nasa.gov/stella/stella-q/stella-q-build-instructions/ includes:

> Use the conductive ink pen and connect the jumpers (A0, A1, A2) on the back of the 4 digit display.
 
This seems to be missing from the assembly instructions at:

> https://landsat.gsfc.nasa.gov/wp-content/uploads/2023/03/STELLA-Q-Part-2-Assembly-Instructions.txt

You may want to update the latter to include the jumper connection instructions. Also maybe indicate that each one is connected to itself by filling in the 'V' shaped gap, just in case anyone tries to connect the three jumpers to each other!

**Reply by paulmirel** [reply]
Is the wall height long enough to mount the microcontroller vertically on the wall, so that the usb port is accessible either through a hole in the lid, or with the lid removed? That way you could cut a hole in the bottom large enough for a finger to get in to release/remove the SD card. Foam double stick tape might be secure enough to make that work. 

Accommodating the keyboard is a good idea. Whatever works to do that is good. 

You can mount the sensors on the inside. The only part that needs to see outside of the box is the sensor itself, which is the little square in the center of each board. The 0.5mm hole a little off center on the sensor is the light aperture, so that's what really needs to see out of the box. A round hole, 1cm or 0.5", should be plenty, as long as you center the sensor on the corresponding hole. A similar hole will allow gas exchange for the air sensor, to allow it to read accurate temperature and humidity

The pencil box is a good idea, but I think smaller might be better. Maybe this jewelry box? https://www.amazon.com/Displays-Earring-Necklace-Jewelry-Boxes-Size/dp/B01N6QTL9D 
Inexpensive, and you can mount the microcontroller on the floor of the box along the near edge, with a hole on either side, for the usb connector on one side, and access to the SD card on the other. Easier to cut holes in cardboard, and it's a more ecologically friendly material, which I think is important for instruments that we hope to inspire some earth science curiosity with. Not quite as nice for the display, which can show right through the lid of the plastic box. We use sticky back velcro squares for mounting components (7/8" size works well), so that we can remove and remount the components easily. 

I like a box design. The stick design is pretty exposed, which, while part of the point to make the workings accessible, also makes it more vulnerable to damage. 

Excellent point about the instructions for closing the jumpers. It's a pleasure working with you!

P

**Reply by mchopping** [reply]
Hi Paul,  Thank you great suggestions!  I will have a go at making some improvements to the first attempt. It will be interesting to see whether the students prefer the case, or the stick versions!

**Reply by mchopping** [reply]
We're going with the plastic case design, since we already have ten cases. I've implemented the first couple of excellent suggestions you made: cut a hole for a USB C port on the bottom (the Feather Express fits perfectly vertically and it's now much easier to extract/insert the SD card); and moved the keyboard inside, affixed to a foam base. A rough diagram is in the attached PDF (I have no skill in technical drawing, but I think I got the gist). One of my students made a suggestion for making the holes using a hot Stanley knife and this works better than the Dremel.  
[STELLA-Q_Montclair_Version_wCase.pdf](https://github.com/STELLA-Landsat/STELLA/files/12907889/STELLA-Q_Montclair_Version_wCase.pdf)

**Reply by paulmirel** [reply]
Excellent! I'm glad you have good accessibility to the port and SD card. Looking forward to some glamour photos!
